daeun/lv1/수박수박수박수박수박수?.js

```diff
@@ -0,0 +1,3 @@
+function solution(n) {
+    return '수박'.repeat(n).slice(0, -n)
+}
```

Comment:
```suggestion
function solution(n) {
    return '수박'.repeat(n).slice(0, -n)
}
```
대운님 진짜 코드 간단하게 효율적으로 잘 짜시는 것 같아요!! 👍
맨날 배워만 가는 것 같지만 또 배워 갑니다~~ 🤗

Reply:
repeat과 slice 생각해보지도 못한 방법이여서 진짜 너무너무 놀랍네요~ 🤔

Reply:
@yjlee9909 이래 보여도 항상 처음 풀면 길이가 길답니다 🥲 항상 좋은 말씀 해주셔서 감사합니다!👍